src/ansys/mapdl/core/cli/hpc.py

```diff
@@ -148,16 +174,6 @@
     flag_value=True,
     help="""Whether the terminal is to wait for job completion before returning control to the user. """,
```

Comment:
Some of these descriptions have a space between the period and closing triple backticks. Do they need it? If so, edits to my earlier suggestions might be necessary.

Reply:
They do not need it.